features/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/themes/appm/js/busunessOwners.js

```diff
@@ -0,0 +1,31 @@
+/*
+ *  Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+$(function () {
+
+    $("#overview_owner").on('click',function() {
+        var names = "";
+        names = $("#owner_list").val();
+        if (names.length > 5) {
```

Comment:
The owner list should be passed here as an Array. Avoid passing the owner list as a string and use of string manipulation to check item availability and owner list retrival